mutox/cli/README.md

```diff
@@ -0,0 +1,102 @@
+# MuTox: MuTox: Universal MUltilingual Audio-based TOXicity Dataset and Zero-shot Detector
+
+MuTox, the first highly multilingual audio-based dataset with toxicity labels.
```

Comment:
And instead of porting this long readme, I would add a small section in the root readme file of this repo, that would point to the full readme in the CS repo. This is because the Mutox paper is already immutable, and it points to the CS repo, so it makes sense to me to keep the detailed Mutox readme there.

@alexmourachko @artemru what do you think?